src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,115 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceTest {
+    private RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+    private StorageDaoImpl storageDao = new StorageDaoImpl();
+
```

Comment:
```suggestion
    private RegistrationServiceImpl registrationService;
    private StorageDaoImpl storageDao;
```
move initialization to a method with annotation BeforeAll or BeforeClass (depends on version)